inference-engine/src/plugin_api/ie_profiling.hpp

```diff
@@ -3,7 +3,8 @@
 //
 
 /**
- * @brief Defines API to profile your plugin using Intel VTune.
+ * @brief [DEPRECATED] Defines API to profile your plugin using Intel VTune.
```

Comment:
looks like we need to add new library documentation to https://docs.openvinotoolkit.org/latest/ie_plugin_api/modules.html

Reply:
Ok, I'll update documentation.